en/docs/guides/access-delegation/get-user-claims-as-a-jwt.md

```diff
@@ -0,0 +1,179 @@
+# Get User Claims as a JWT
+
+This page provides instructions on how to get user claims of the authorized user as a [JWT token](insertlink) with the validation response. 
+
+You can configure this for all tenants using the file configurations by configuring the `deployment.toml` file or you can configure it tenant-wise using the management console. 
```

Comment:
You can configure this for all the tenants by configuring the `deployment.toml` file or configure it tenant-wise using the management console.